src/tests/test_1_annotation.py

```diff
@@ -0,0 +1,159 @@
+from operator import contains
```

Comment:
I don't see `contains` is used anywhere?

Reply:
Line removed. Thank you :)